standup-helper.sh

```diff
@@ -5,6 +5,14 @@ set -e
 # and run git-standup there directly from node (cwd did nothing), so
 # here we are.
 
+if ! type "git-standup" > /dev/null; then
+  echo "Seems like you haven't installed git-standup yet."
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    echo "Installing git-standup with homebrew..."
+    brew install git-standup
```

Comment:
I don't think we can assume clients have homebrew installed. At least we should suggest homebrew installation if the script fails to find it.